src/Miew.js

```diff
@@ -2531,7 +2543,15 @@ Miew.prototype.repGet = function (index, name) {
  */
 Miew.prototype.repAdd = function (rep, name) {
   const visual = this._getComplexVisual(name);
-  return visual ? visual.repAdd(rep) : -1;
+  if (!visual) {
```

Comment:
What about addition by selection (X button)? Will the event fire in the case?

Reply:
Adding by selection does not use this function. But I also put dispatching correspondent event in _extractRepresentation() (line 2423 miew.js)